README.adoc

```diff
@@ -25,16 +25,18 @@ Learn how to use MicroProfile Rest Client to invoke RESTful microservices over H
 
 == What you'll learn
 
-You will learn how to build a MicroProfile Rest Client to access remote RESTful services.
-MicroProfile Rest Client provides a type-safe way to call RESTful services so that you can focus more on the client services rather than how to connect with remote services.
+You will learn how to build a MicroProfile Rest Client to access remote RESTful services. MicroProfile Rest Client provides a type-safe way to invoke RESTful services.
 
 The application that you will be working with is an `inventory` service, which fetches and stores the system property information for different hosts.
 Whenever a request is made to retrieve the system properties of a particular host, the `inventory` service will create a client to invoke the `system`
 service on that host. The `system` service simulates a remote service in the application.
 
-You will learn how to create and register the client interface and handle exceptions through the `ResponseExceptionMappers` mapper class.
-Then, you will instantiate the client and use it in the `inventory` service. You can choose from two different approaches: CDI with the help of MicroProfile Config or the `RestClientBuilder` method.
-In this guide, you will explore both methods in order to handle different scenarios to provide a valid base URL. You need a valid base URL to instantiate the RESTful client.
+You will learn how to create a template interface that maps to the remote service that you want to call.
+The MicroProfile Rest Client API automatically generates an instance of the template interface based on its annotations, so that you don't waste time on setting up the client and connecting with the remote service.
+
+You will instantiate the client and use it in the `inventory` service. You can choose from two different approaches: Context and Dependency Injection (CDI) with the help of MicroProfile Config, or the `RestClientBuilder` method.
```

Comment:
The colon looks more formal, I think. We also don't need a comma after "Config" because serial commas apply to three or more items. The "or" is sufficient here without a comma. :) So how about this...

"You can choose from two different approaches, Context and Dependency Injection (CDI) with the help of MicroProfile Config or the `RestClientBuilder` method."

It's typical to put the longer item last, so if you want, you could mention the `RestClientBuilder` method first.